### ISSUE TITLE
Re-ordering entries causes multiple commits

### ISSUE THREAD

**Issue by jesseleite** [statamic/spock#33]
Re-ordering entries causes multiple commits (one per entry), in contrast to re-ordering pages which fires a single `PagesMoved` event (and therefore a single commit).

This actually needs to be addressed in the Statamic core with a new `EntriesMoved` event that overrides the individual `EntrySaved` events, but since it's being reported in unrelated issues here (see https://github.com/statamic/spock/issues/32#issuecomment-452055537 and https://github.com/statamic/spock/issues/31#issuecomment-453615468), I thought I'd create a new issue to address this specifically 🙂

**Comment by timichango** [comment]
Hey Jesse — reckon this is a dupe/related to my other issue posted a few days ago, #32

**Comment by timichango** [comment]
Or... wait, is it?

**Comment by timichango** [comment]
Sorry, my bad, originally was confused and thought your comment mentioning me was on that other thread (the request someone else made for change count thresholds)

**Comment by jesseleite** [comment]
Heyo!  Adding a file limit (#31), and git hanging/performance (#32), are separate issues (though I know you mentioned the entry re-ordering thing in both of those threads).

We currently fire a single event/commit when re-ordering pages 👍, but not yet when re-ordering entries 👎 (hence the creation of this issue, to recognize this specifically as it's own bug/oversight) 👍

**Comment by timichango** [comment]
Agreed re. file limit stuff (that wasn't my idea / I don't have a dog in that particular fight, so to speak) — was only commenting in that thread re. the multiple commits happening on reorder, on account of the broader commonality of both circumstances leading to a bloat of the commits. And it never once occurred to me that there was any relationship between the spock/git/hang issue and the notion of some sort of file count threshold.

See my comments on #32 re. the spock/reorder/hang thing (responded to a prior inquiry of yours in that thread earlier). 

Cheers!

**Comment by jesseleite** [comment]
Haha no worries man!  Just trying to separate these issues out so we can properly deal with them.  Take care!

**Comment by pxwee5** [comment]
This should be critical. I'm running into issues where:
- I have 70 press releases
- The 71st press release is created under the name `71.new name`
- Editor sometimes re-orders it to the first position. 
- This means 70 file renames 
- And then Spock commits them 1 by 1, **SEVENTY** times.

**Comment by jackmcdade** [comment]
This is addressed in Statamic v3, coming pretty soon. Page trees are stored
in a single file with references to their IDs, which means only one file
change, and one commit!
